jsx/Form.js

```diff
@@ -1736,6 +1736,138 @@ class LorisElement extends Component {
+      ? parseFloat(e.target.value)
+      : this.props.valueMin;
+    let value = inputValue > this.props.valueMax
+      ? this.props.valueMax
+      : inputValue;
+    value = value < this.props.valueMin
+      ? this.props.valueMin
+      : value;
+    this.props.onUserInput(this.props.name, value);
+  }
+
+  render() {
+    let errorMessage = null;
+    let requiredHTML = null;
+    let elementClass = this.props.elementClass;
+    let disabled = this.props.disabled ? 'disabled' : null;
+    let required = this.props.required ? 'required' : null;
+    // Add required asterix
+    if (required) {
+      requiredHTML = <span className='text-danger'>*</span>;
+    }
+    // Add error message
+    if (this.props.errorMessage) {
+      errorMessage = <span>{this.props.errorMessage}</span>;
+      elementClass = this.props.elementClass + ' has-error';
+    }
+
+    return (
+      <div className={elementClass}>
+        <label className={'col-sm-3 control-label'}
+               htmlFor={this.props.id}>
+          {this.props.label}
+          {errorMessage}
+          {requiredHTML}
+        </label>
+        <div className={'col-sm-9'}>
+          <div style={{
+            display: 'flex',
+            flexDirection: 'row',
+            flexWrap: 'wrap',
+            width: '100%',
+          }}>
+            <div style={{
+              display: 'flex',
+              flexDirection: 'column',
+              flexBasis: '100%',
+              maxWidth: '50px',
+              flex: 1,
+            }}>
+              <input
+                type='number'
+                name={'input_' + this.props.name}
+                value={this.props.value}
+                min={this.props.valueMin}
+                max={this.props.valueMax}
+                required={required}
+                disabled={disabled}
+                onChange={this.handleChange}
+                style={{
+                  width: '50px',
+                  textAlign: 'center',
+                }}
+              />
+            </div>
+            <div style={{
+              flexGrow: 1000,
+              display: 'flex',
+              flexDirection: 'column',
+              flexBasis: '100%',
+              flex: 2,
+            }}>
+              <input
+                type='range'
+                name={this.props.name}
+                id={this.props.id}
+                value={this.props.value}
+                min={this.props.valueMin}
+                max={this.props.valueMax}
+                required={required}
+                disabled={disabled}
+                onChange={this.handleChange}
+                style={{width: '100%'}}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+SliderElement.propTypes = {
+  id: PropTypes.string,
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  value: PropTypes.number.isRequired,
+  valueMin: PropTypes.number.isRequired,
```

Comment:
could we change to minValue and maxValue? or even just min and max to keep consistent with props of other elements

Reply:
@zaliqarosli I originally had min & max. I thought it would be better since valueMin & valueMax are more coherent to the min & max are being used for value. It doesn't bother me but I'm curious what you think with the previous said. I know you use vim but as a developer with an IDE if you start typing "value" all the other prop types with value show up and helps in aiding what else. In any case I don't really mind.

Reply:
if you don't mind, i would prefer minValue and maxValue please. thank you!